HowTos/controller_migration.rst

```diff
@@ -101,4 +101,17 @@ Workflow
 2. New controller perform "Settings->Maintenance->Backup&Restore->Restore->Restore(with latest backed up file)
 3. If you want to keep the old controller public ip, detach it from the old controller and reattch to new controller. Otherwise perform "Troubleshoot->Diagnostics->Network->Controller IP Migration->Migrate"
 
+Controller Migration in GCP
+##################################
+GCP controller image in 5.3 and previous releases is based of 14.4 ubuntu distribution. 5.4 versions and higher versions of controller image will be based of 18.4 ubuntu distribution. Controller upgrade from 5.3 to 5.4 is not supported, instead the following workflow needs to be used.
+
+Controller Migration from 5.3 to 5.4
+1. On old controller (with version 5.3) perform "Settings->Maintenance->Backup&Restore->Backup->Backup Now"
```

Comment:
Somehow the numbered list 1 and 2 are in one line. Need to end a new line between them